presto-jdbc/src/test/java/io/prestosql/jdbc/BaseTestJdbcResultSet.java

```diff
@@ -170,10 +170,10 @@ public void testTime()
                 assertEquals(rs.getObject(column, Time.class), Time.valueOf(LocalTime.of(9, 39, 5)));
                 assertThatThrownBy(() -> rs.getDate(column))
                         .isInstanceOf(SQLException.class)
-                        .hasMessage("Invalid date from server: 09:39:05.000"); // TODO (https://github.com/prestosql/presto/issues/5313) wrong message
+                        .hasMessage("Expected value to be a date but is : 09:39:05.000");
                 assertEquals(rs.getTime(column), Time.valueOf(LocalTime.of(9, 39, 5)));
                 assertThatThrownBy(() -> rs.getTimestamp(column))
-                        .isInstanceOf(IllegalArgumentException.class) // TODO (https://github.com/prestosql/presto/issues/5315) SQLException
```

Comment:
This PR is about `.getDate`'s exception _message_ and the linked issue is about `.getTimestamp`'s exception _type_.

Why the TODO is removed? I guess it was not intentional, please revert.